packages/core/getBusyTimes.ts

```diff
@@ -210,4 +210,63 @@ export async function getBusyTimes(params: {
   return busyTimes;
 }
 
+export async function getBusyTimesForLimitChecks(params: {
+  userId: number;
+  eventTypeId: number;
+  startDate: Date;
+  endDate: Date;
+}) {
+  const { userId, eventTypeId, startDate, endDate } = params;
+  logger.silly(
+    `Fetch limit checks bookings in range ${startDate} to ${endDate} for input ${JSON.stringify({
+      userId,
+      eventTypeId,
+      status: BookingStatus.ACCEPTED,
+    })}`
+  );
+  performance.mark("getBusyTimesForLimitChecksStart");
+
+  const bookings = await prisma.booking.findMany({
+    where: {
+      userId,
+      eventTypeId,
+      status: BookingStatus.ACCEPTED,
+      // FIXME: bookings that overlap on one side will never be counted
+      startTime: {
+        gte: startDate,
+      },
+      endTime: {
+        lte: endDate,
+      },
+    },
```

Comment:
Should probably get all bookings that start in that period, not also end. Need some input here.

Reply:
True, an event end time typically does not matter unless it's greater than startDate - this probably makes the most sense.

```suggestion
  const bookings = await prisma.booking.findMany({
    where: {
      userId,
      eventTypeId,
      status: BookingStatus.ACCEPTED,
      // FIXME: bookings that overlap on one side will never be counted
      startTime: {
        gte: startDate,
      },
      endTime: {
        gte: startDate,
      },
    },
```